src/main/java/com/conveyal/r5/analyst/broker/BrokerHttpHandler.java

```diff
@@ -65,6 +66,10 @@ public void service(Request request, Response response) throws Exception {
         if (pathComponents.length < 2) {
             response.setStatus(HttpStatus.BAD_REQUEST_400);
             response.setDetailMessage("path should have at least one part");
+            TaskError error = new TaskError(null, "Path should have at least one part", "");
```

Comment:
Then again I see that you've done this throughout the class, so I guess the idea is to change the purpose/meaning of TaskError to represent all errors.